src/Views/BasicNumbersInput.js

```diff
@@ -0,0 +1,50 @@
+import { Console } from '@woowacourse/mission-utils';
+import throwError from '../Utils/throwError.js';
+import Rules from '../Utils/Rules.js';
+import Errors from '../Constants/Errors.js';
+import { InputComment } from '../Constants/display.js';
+
+const BasicNumbersInput = {
+  get: async () => {
+    const userInput = await Console.readLineAsync(InputComment.BASIC_NUMBERS);
+    return userInput;
+  },
+
+  validate: (basicNumbersInput) => {
+    if (Rules.isNoValueString(basicNumbersInput))
+      throwError(Errors.BasicNumbers.NO_INPUT);
+
+    const basicNumbers = basicNumbersInput.split(',');
+    if (Rules.isWrongLength(basicNumbers, 6))
+      throwError(Errors.BasicNumbers.IS_WRONG_LENGTH);
+    if (Rules.isDuplicatedValue(basicNumbers))
+      throwError(Errors.BasicNumbers.IS_DUPLICATED_VALUE_IN);
+
+    return validateEachNumber(basicNumbers);
+  },
+
+  parse: (basicNumbersInput) => {
+    return basicNumbersInput
+      .split(InputComment.Seperator.VALUE)
+      .map((numberString) => Number(numberString));
```

Comment:
사소하지만 이렇게 바로 적용하는 방식은 어떤가요? 개인적으로 함수 참조방식을 좋아해서요ㅎㅎ
```suggestion
      .split(InputComment.Seperator.VALUE)
      .map(Number);
```

Reply:
아예 함수 이름만 넣으면 적용해서 새로운 배열을 만들어주나 보네요! 몰랐던 사실이에요 🥲
`map`함수를 써볼 때 신경 써보면 너무 좋을 것 같아요 감사합니다!

해성님도 프리코스 고생많으셨습니다🫠🫠
4주차 미션이 저도 너무 어려워서..🥲 한 달간 준비 열심히 해봐야겠어요. 화이팅입니다!